docs/Catalog/FileBrowser.md

```diff
@@ -0,0 +1,33 @@
+## Files manipulation in Catalog
+
+### Files browser
+
+Every bucket has "Bucket" tab where the user can see all files in his bucket.
+
+![Files browser tab](../imgs/catalog-filesbrowser-tab.png)
+
+See also [docs on how to hide this tab](./Preferences.md).
+
+### Bookmarks
+
+If you want to create a package including multiple files from different directories and even different buckets, you can browse files and bookmark needed files. You can select needed files and click "Add to bookmarks". You can also navigate to a specific file and bookmark an individual file from that file's page.
+
+![Select and add to bookmarks](../imgs/catalog-filesbrowser-addtobookmarks.png)
+
+Then you can open the Bookmarks pane and a create package from all those bookmarks.
+
+![Open bookmarks](../imgs/catalog-filesbrowser-bookmarksmenu.png)
+
+![Browse bookmarks](../imgs/catalog-filesbrowser-bookmarkspane.png)
+
+### Text editor
+
+You can edit plain text, Markdown, JSON and YAML files.
```

Comment:
How do I edit an existing file? Is there a special button, like there is for a new file? Is there an image demonstrating that?

With that fix (and the other grammar changes I made), I think this will be good to go.

Reply:
Added screenshot of a file page. There is an "edit" button